000-cbdb-sandbox/README.md

```diff
@@ -102,5 +122,63 @@ In addition to using the `docker exec` command, you can also use the `ssh` comma
 ssh gpadmin@localhost # Password: cbdb@123
 ```
 
-Now you have a Cloudberry Database and can continue with [Cloudberry Database Tutorials Based on Single-Node Installation](https://github.com/cloudberrydb/bootcamp/blob/main/101-cbdb-tutorials/README.md)! Enjoy!
+Now you have a Cloudberry Database and can continue with [Cloudberry Database Tutorials Based on Docker Installation](https://github.com/cloudberrydb/bootcamp/blob/main/101-cbdb-tutorials/README.md)! Enjoy!
+
+## Working with your Cloudberry Docker environment
+
+When working with the Cloudberry Docker environment there are a few commands that will be useful to you.
+
+**Stopping Your Single Container Deployment With Docker**
+
+To stop the **single container** deployment while _keeping the data and state_ within the container, you can run the command below. This means that you can later start the container again and any changes you made to the containers will be persisted between runs.
+
+```shell 
+docker stop cbdb-mdw
+```
+
+To stop the **single container** deployment and also remove the volume that belongs to the container, you can run the following command. Keep in mind this will remove the volume as well as the container associated which means any changes you've made inside of the container or any database state will be wiped and unrecoverable.
+
+```shell 
+docker rm -f cbdb-mdw
+```
+
+**Stopping Your Multi-Container Deployment With Docker**
+
+To stop the **multi-container** deployment while _keeping the data and state_ within the container, you can run the command below by subsituting the <docker-compose-OS.yml> argument with whichever docker-compose file is associated with your deployment (Rocky Linux 9 or CentOS). This means that you can later start the container again and any changes you made to the containers will be persisted between runs.
+
+```shell 
+docker compose -f <docker-compose-OS.yml> stop
+```
+
+To stop the **multi-container** deployment while _keeping the data and state_ within the container, you can run the command below by subsituting the <docker-compose-OS.yml> argument with whichever docker-compose file is associated with your deployment (Rocky Linux 9 or CentOS). Running this command means it will delete the containers as well as remove the volumes that the containers are associated with. This means any changes you've made inside of the containers or any database state will be wiped and unrecoverable. 
+
+```shell 
+docker compose -f <docker-compose-OS.yml> down
+```
```

Comment:
I believe the initial sentence is incorrect as the `down` operation will delete the containers. The sentence indicates the data and state are kept.

Reply:
You're correct on this, looks like it was maybe a copy-paste that didn't get corrected. Fixed below, also added the -v flag which I started using today since otherwise it'll leave you with dangling volumes that can't be reattached later if you use down
https://github.com/antoniopetrole/bootcamp/commit/a8fb88a41f3de46245e5860d086a8362e5d3061f